typescript/solver/config.ts

```diff
@@ -0,0 +1,15 @@
+import dotenvFlow from "dotenv-flow";
```

Comment:
Probably would be good eventually to split this up into general solver config (pkey) and intent module specific config (contract addresses)

Reply:
Created issue: https://github.com/BootNodeDev/intents-framework/issues/18